scripts/core/event.yml

```diff
@@ -75,6 +75,10 @@ etors:
       desc: Event created by $xEnqueueWriteHostPipe
     - name: COMMAND_BUFFER_ENQUEUE_EXP
       desc: Event created by $xCommandBufferEnqueueExp
+    - name: INTEROP_SEMAPHORE_WAIT_EXP
+      desc: Event created by $xBindlessImagesWaitExternalSemaphoreExp
+    - name: INTEROP_SEMAPHORE_SIGNAL_EXP
+      desc: Event created by $xBindlessImagesSignalExternalSemaphoreExp
```

Comment:
These should defined `value:`s at an offset, e.g. `0x1000`, to avoid conflicting with the core spec enumeration range.